src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
     }
```

Comment:
Don't use `toString()` for getting `String` representation of enum constants. Use `name()` instead. It's a common practice to use `name()` for this purpose because `toString()` can be overridden and start returning not the name of the enum constant. So to avoid such situations we use `name()`. Your code should look like this: `return color.name();`